api/MediaStreamEvent.json

```diff
@@ -0,0 +1,166 @@
+{
+  "api": {
+    "MediaStreamEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaStreamEvent",
+        "support": {
+          "webview_android": {
+            "version_added": true
+          },
+          "chrome": {
+            "version_added": true
+          },
+          "chrome_android": {
+            "version_added": true
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": true
+          },
+          "firefox_android": {
+            "version_added": null
+          },
+          "ie": {
+            "version_added": false
+          },
+          "nodejs": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": true
+          },
+          "opera_android": {
+            "version_added": true
+          },
+          "safari": {
+            "version_added": null
+          },
+          "safari_ios": {
+            "version_added": null
+          },
+          "samsunginternet_android": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": false,
+          "deprecated": true
+        }
+      },
+      "MediaStreamEvent": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaStreamEvent",
```

Comment:
This should be <code>"mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaStreamEvent/MediaStreamEvent"</code>, also there should be a `"description": "<code>MediaStreamEvent()</code> constructor"` entry.